doc/tokens/tokentypes/daypassword.rst

```diff
@@ -0,0 +1,43 @@
+.. _daypassword_token:
+
+Day Password Token
+-------------------
+
+The day password token is a modification of the :ref:`totp_token`. The difference between the
+:ref:`totp_token` and day password token is, that the PIN of the day password token can be used more
+then ones. For example you can use the day password token as a day password, in this way you get every
+day a new password for the day.
+Because the day password token basing on :ref:`totp_token` it works with the same Algorithms. You
+can find the Algorithm hir `RFC6238 <https://tools.ietf.org/html/rfc6238>`_.
+
+The day password token algorithm has some parameter, like if the generated OTP value will
```

Comment:
The parameters should be described under "Enrollment"

Reply:
https://github.com/privacyidea/privacyidea/pull/3602/commits/1d33e4a5414b7c3962159450ba40a6e9f9e29488